docs/developers.rst

```diff
@@ -0,0 +1,173 @@
+
+.. inclusion-marker-start-do-not-remove
+
+
+Developer Guide
```

Comment:
What do you think about calling it "Contribution Guide"?

Reply:
Changed to "Contributor Guide".  What do you think?  (I prefer Contributor over Contribution as the former is more broad, meaning we can include stuff useful to contributors beyond explaining how to make contributions)

Reply:
That works!